apps/cookbook/src/app/examples/item-example/examples/input/default.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-item-example-input-default',
+  template: `<kirby-item>
+  <kirby-avatar slot="start">
+    <kirby-icon name="moneybag"></kirby-icon>
+  </kirby-avatar>
+  <h3>Title</h3>
+  <kirby-form-field slot="end">
+    <input type="text" kirby-input value="Tekst kan skrives her"/>
```

Comment:
Please align language with rest of cookbook examples (english)
```suggestion
    <input type="text" kirby-input value="Item with input"/>
```